src/css/bundle.scss

```diff
@@ -288,6 +300,7 @@ input[type='text'].display {
      */
     .form-group {
         label {
+            color: white;
```

Comment:
Check the _variables.scss $white variable and use these for colors. 
This allows us to easily switch out colors, when needed. A high level example would be a possible light mode theme, which would need all colors / backgrounds to be set by variables.